{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "   selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started\n   Stacktrace:\n   NoSuchDriverError: A session is either terminated or not started\n       at asyncHandler (/usr/local/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/protocol.js:311:15)\n       at /usr/local/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/protocol.js:514:15\n       at Layer.handle [as handle_request] (/usr/local/lib/node_modules/appium/node_modules/express/lib/router/layer.js:95:5)\n       at next (/usr/local/lib/node_modules/appium/node_modules/express/lib/router/route.js:149:13)\n       at Route.dispatch (/usr/local/lib/node_modules/appium/node_modules/express/lib/router/route.js:119:3)\n       at Layer.handle [as handle_request] (/usr/local/lib/node_modules/appium/node_modules/express/lib/router/layer.js:95:5)\n       at /usr/local/lib/node_modules/appium/node_modules/express/lib/router/index.js:284:15\n       at param (/usr/local/lib/node_modules/appium/node_modules/express/lib/router/index.js:365:14)\n       at param (/usr/local/lib/node_modules/appium/node_modules/express/lib/router/index.js:376:14)\n       at Function.process_params (/usr/local/lib/node_modules/appium/node_modules/express/lib/router/index.js:421:3)\n       at next (/usr/local/lib/node_modules/appium/node_modules/express/lib/router/index.js:280:10)\n       at logger (/usr/local/lib/node_modules/appium/node_modules/morgan/index.js:144:5)\n       at Layer.handle [as handle_request] (/usr/local/lib/node_modules/appium/node_modules/express/lib/router/layer.js:95:5)\n       at trim_prefix (/usr/local/lib/node_modules/appium/node_modules/express/lib/router/index.js:328:13)\n       at /usr/local/lib/node_modules/appium/node_modules/express/lib/router/index.js:286:9\n       at Function.process_params (/usr/local/lib/node_modules/appium/node_modules/express/lib/router/index.js:346:12)\n       at next (/usr/local/lib/node_modules/appium/node_modules/express/lib/router/index.js:280:10)\n       at /usr/local/lib/node_modules/appium/node_modules/body-parser/lib/read.js:137:5\n       at AsyncResource.runInAsyncScope (node:async_hooks:211:14)\n       at invokeCallback (/usr/local/lib/node_modules/appium/node_modules/raw-body/index.js:238:16)\n       at done (/usr/local/lib/node_modules/appium/node_modules/raw-body/index.js:227:7)\n       at IncomingMessage.onEnd (/usr/local/lib/node_modules/appium/node_modules/raw-body/index.js:287:7)\n", "test_name": "test_valid_login", "rerun": "0"}}, "suite_name": "tests/test_login.py"}}}, "date": "March 03, 2025", "start_time": 1741004281.612179, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "1", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "1"}